src/main/logutils.ts

```diff
@@ -110,48 +165,153 @@ const tailFile = (path: string) => {
     tailHandler.on("line", function(data: string) {
         handleLogLine(data);
     });
-    
+
     tailHandler.on("error", function(error: unknown ) {
       console.log('ERROR: ', error);
     });
-}    
+}
 
 /**
+ * Splits a WoW combat line intelligently with respect to quotes,
+ * lists, tuples, and what have we.
+ *
+ */
+const splitLogLine = (line: string, maxSplits?: number): LogLine => {
+    const line_len = line.length
+    const list_items: string[][] = [];
+    const args_list: any[] = [];
+    let in_quote = false;
+    let open_lists = 0;
+    let value: any = '';
+
+    // Combat log line always has '<timestamp>  <line>' format,
+    // that is, two spaces between ts and line.
+    const tsIndexEnd = line.indexOf('  ');
+    const timestamp = line.substring(0, tsIndexEnd);
+
+    for (let ptr = tsIndexEnd + 2; ptr < line_len; ptr++) {
+        const c = line.charAt(ptr);
+        if (c === '\n') {
+            break;
+        }
+
+        if (in_quote) {
+            if (c === '"') {
+                in_quote = false;
+                continue;
+            }
+
+        } else {
+            switch (c) {
+            case ',':
```

Comment:
indentation a bit off? a few lines below too

Reply:
Yeah, it looks like it, but VSCode likes `case` to be at the same indentation level as `switch`. I guess it's too avoid a too high level of intendation (one extra level).